### ISSUE TITLE
Adding first role fails without config file

### ISSUE THREAD

**Issue by jfexyz** [jbuck/assume-aws-role#2]
This error is thrown:

```
fs.js:584
  return binding.open(pathModule._makeLong(path), stringToFlags(flags), mode);
                 ^

Error: ENOENT: no such file or directory, open '/Users/{user}/.assume-aws-role/config'
    at Error (native)
    at Object.fs.openSync (fs.js:584:18)
    at Object.fs.readFileSync (fs.js:431:33)
    at Object.readFileSync (/usr/local/lib/node_modules/assume-aws-role/node_modules/jsonfile/index.js:31:20)
    at Object.<anonymous> (/usr/local/lib/node_modules/assume-aws-role/bin/assume-aws-role.js:40:19)
    at Module._compile (module.js:398:26)
    at Object.Module._extensions..js (module.js:405:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Function.Module.runMain (module.js:430:10)
```

**Comment by jbuck** [comment]
Huh, that's not supposed to happen!

Looks like I didn't read the documentation correctly, [readJsonSync](https://github.com/jprichardson/node-fs-extra#readjsonfile-options-callback) with `throws: false` only prevents a JSON parsing error from being thrown, not an fs error. Adding a try/catch around this should do the trick... maybe even just go back to the plain old fs module.

**Comment by jbuck** [comment]
@joshuajabbour I've published assume-aws-role@1.1.1 that includes error handling for the ENOENT error. The cause of this error was an older version of a dependency I had installed that handled the ENOENT error by itself. Please install it and let me know whether it's fixed this issue for you!

**Comment by jfexyz** [comment]
Yep, fixed. Thanks!
